### DISCUSSION TITLE
How to configure an HTTPS proxy?

### DISCUSSION THREAD

**Discussion by Jayasheel** [eclipse-edc/Connector#734] [Q&A] [answered]
My build server is behind firewall and needs to go through a Proxy server. gradlew -> gradle-wrapper.jar apparently does not have a way to specify a proxy server and build is stuck at:
Downloading https://services.gradle.org/distributions/gradle-7.1.1-bin.zip

**Comment by MoritzKeppler** [comment, marked as answer]
have you tried to configure the proxy as described in
https://docs.gradle.org/current/userguide/gradle_wrapper.html and 
https://docs.gradle.org/current/userguide/build_environment.html#sec:accessing_the_web_via_a_proxy

<sub> Moritz Keppler <moritz.keppler@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>
